moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -848,25 +851,26 @@ shapes::Mesh* VisibilityConstraint::getVisibilityCone(const robot_state::RobotSt
 }
 
 void VisibilityConstraint::getMarkers(const robot_state::RobotState& state,
-                                      visualization_msgs::MarkerArray& markers) const
+                                      visualization_msgs::msg::MarkerArray& markers)
 {
   shapes::Mesh* m = getVisibilityCone(state);
-  visualization_msgs::Marker mk;
+  visualization_msgs::msg::Marker mk;
   shapes::constructMarkerFromShape(m, mk);
+  rclcpp::Time stamp = clock_ros_.now();
   delete m;
   mk.header.frame_id = robot_model_->getModelFrame();
-  mk.header.stamp = ros::Time::now();
+  mk.header.stamp = stamp;
```

Comment:
Why not use `clock_ros_.now()` directly? I don't see `stamp` being used anywhere else.

Reply:
Changes were made on the following commit https://github.com/ros-planning/moveit2/commit/b3855c3fb738fad92a5a8f4ae74c5068c23620aa